rosa_hcp/rosa-hcp-aws-privatelink-creating-cluster.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-hcp-aws-privatelink-creating-cluster"]
+= Creating a private cluster on {hcp-title}
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-aws-privatelink-creating-cluster
+
+toc::[]
+
+This document describes how to create a {hcp-title-first} private cluster.
+
+include::modules/osd-aws-privatelink-about.adoc[leveloffset=+1]
+include::modules/osd-aws-privatelink-required-resources.adoc[leveloffset=+1]
+include::modules/rosa-hcp-aws-privatelink-create-cluster.adoc[leveloffset=+1]
+include::modules/rosa-hcp-aws-privatelink-security-groups.adoc[leveloffset=+1]
+
+== Next steps
+xref:../rosa_install_access_delete_clusters/rosa-sts-config-identity-providers.adoc#rosa-sts-config-identity-providers[Configure identity providers]
+
+[role="_additional-resources"]
+== Additional resources
+
+* xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs[AWS PrivateLink firewall prerequisites]
+* xref:../rosa_getting_started/rosa-sts-getting-started-workflow.adoc#rosa-sts-overview-of-the-deployment-workflow[Overview of the ROSA with STS deployment workflow]
+* xref:../rosa_install_access_delete_clusters/rosa-sts-deleting-cluster.adoc#rosa-sts-deleting-cluster[Deleting a ROSA cluster]
+* xref:../rosa_architecture/rosa_architecture_sub/rosa-architecture-models.adoc#rosa-architecture-models[ROSA architecture]
```

Comment:
I guess technically, even though it's in the ROSA architecture section, the destination page itself is called "Architecture models":
```suggestion
* xref:../rosa_architecture/rosa_architecture_sub/rosa-architecture-models.adoc#rosa-architecture-models[Architecture models]
```